src/shared/api.ts

```diff
@@ -67,3 +68,56 @@ export const getMovies = async (
 
   return await dataFetch();
 };
+
+export const createMovie = async (movie: IMovie) => {
+  const dataFetch = async () => {
+    try {
+      const response = await fetch(`${moviesApiBaseUrl}`, {
+        method: 'POST',
+        body: JSON.stringify(movie),
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+      if (!response.ok) {
+        throw new Error('Internal Error');
+      }
+      return { status: 'ok' };
```

Comment:
I suggest returning the whole movie description or at least a movie ID to be able to redirect to movie details. Same to the next function.